akka-docs/src/main/paradox/serialization-jackson.md

```diff
@@ -377,7 +377,9 @@ the `jackson-json` binding the default configuration is:
 
 @@snip [reference.conf](/akka-serialization-jackson/src/main/resources/reference.conf) { #compression }
 
-Messages larger than the `compress-larger-than` property are compressed with GZIP.
+Supported compression algorithms are: gzip, lz4. Use 'off' to disable compression.
+Gzip is builtin and slower than lz4. For lz4 compression, [lz4-java](https://github.com/lz4/lz4-java) is required.
```

Comment:
is required, but it is already a transitive dependency of `akka-serialiation-jackson`.